models/staging/stg_pardot.yml

```diff
@@ -262,10 +269,11 @@ models:
   - name: stg_pardot__opportunity_prospect
     description: Each record represents a prospect's involvement in an opportunity in Pardot.
     columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
       - name: opportunity_prospect_id
         description: Primary key for the table
         tests:
-          - unique
```

Comment:
We can revert and bring this unique test back in, as `source_relation` is being included in the surrogate key now.